.github/workflows/publish.yml

```diff
@@ -18,6 +18,9 @@ jobs:
           pnpm install
           pnpm build
 
+      - name: Reset uncommitted changes
+        run: git checkout -- .
```

Comment:
why is this needed? there should not be any uncommitted changes resulting from just running install and build.

Reply:
Removed it.